### DISCUSSION TITLE
为什么程序在一台电脑上内存占用是200MB，到了另一台电脑占用直接5g

### DISCUSSION THREAD

**Discussion by hhfwork** [BYJRK/DotNet-Discussions#136] [Q&A]
我写了很简单的程序，字符串分割。两个textbox和一个button，在自己电脑上内存占用150MB，复制到另一台电脑上占用5g。
写的肯定定是没问题的，dll只用了CommunityToolkit.Mvvm，HandyControls。操作都用的是stringbuilder。
自己的电脑比较好，win10x64，内存32g，显卡4060，
另一台电脑比较老了、10年有了，win7x64，内存8g，集显，日常打开其他软件也都是正常的，就这个软件直接5g。
我将release版本拷贝到另一台电脑，只是打开还没任何操作。但凡开两个，另一台电脑就卡死了。
问一下大神们，这可能什么原因，或者能不能提供一个思路。

**Comment by hhfwork** [comment]
[解决].net7.0改成.net6.0，搞定

**Reply by BYJRK** [reply]
.NET 7 这么离谱的吗😂手头没有 Win 7 系统的电脑，以后有时间试一下。
